lib/test.py

```diff
@@ -0,0 +1,238 @@
+import os
+import time
+from enum import Enum
+from typing import Union
+
+import pytest
+from openai import pydantic_function_tool
+from pydantic import BaseModel, Field
+
+from lib.models import GigaChatModel, MistralModel, OpenAIModel
```

Comment:
Если файл в директории lib, то просто models
```suggestion
from models import GigaChatModel, MistralModel, OpenAIModel
```

Reply:
поправил